episodes/24-collaborating.md

```diff
@@ -100,7 +100,7 @@ The moment your collaborator group exceeds 3 or 4 members (and fewer if the coll
 
 Here are some aspects of governing a project that you should consider. These are borrowed from [the _Working in teams_ chapter](https://merely-useful.tech/py-rse/teams.html) of [_Research Software Engineering with Python_](https://merely-useful.tech/py-rse/index.html), a book on how to work productively in small teams where everyone is welcome:
 
-- [establish a Code of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
+- [establishing a Code of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
```

Comment:
```suggestion
- [Codes of Conduct](https://merely-useful.tech/py-rse/teams.html#teams-coc)
```

I think this is more consistent with the phrasing of the other points in the list.